### ISSUE TITLE
Разные цветные темы

### ISSUE THREAD

**Issue by OlegWock** [filingroove/vk-prism#1]
Как на счет добавить хоть несколько тем на выбор. Хотя бы те же monokai и гитхабовскую

**Comment by filingroove** [comment]
Да, это хорошая идея :+1: Это не сложно реализовать. Займусь как только найду минутку :)

**Comment by OlegWock** [comment]
Спасибо
